starter-code/public/javascript/APIHandler.js

```diff
@@ -1,25 +1,63 @@
 class APIHandler {
-  constructor (baseUrl) {
+  constructor(baseUrl) {
     this.BASE_URL = baseUrl;
   }
 
-  getFullList () {
-
+  getFullList() {
+    const url = this.BASE_URL;
+    return axios
+      .get(`${url}/characters`)
+      .then(response => {
+        // console.log(response.data);
+        let result = response.data;
+        return result;
+      })
+      .catch(err => console.log(err));
   }
 
-  getOneRegister () {
-
+  getOneRegister(id) {
+    return axios
+      .get(`http://localhost:8000/characters/${id}`)
+      .then(response => {
+        return response.data;
+      })
+      .catch(err => console.log(err));
   }
 
-  createOneRegister () {
-
+  createOneRegister(name, occupation, weapon, cartoon) {
+    axios
+      .post("http://localhost:8000/characters", {
+        name: name,
+        occupation: occupation,
+        weapon: weapon,
+        cartoon: cartoon
+      })
+      .then(response => {
+        console.log(response.data);
```

Comment:
even if you don't need the response for now, try to keep the habit of making all the methods consistent with their return values